5.5/examples/replica/mysql_replica.json

```diff
@@ -68,6 +68,23 @@
       }
     },
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "mysql-slave"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "512Mi"
```

Comment:
Is this template meant to be custom-made for OSO?

The master has a claim of 512 MiB, that's why I picked the same number.
The claim being of 512 MiB doesn't require the PVs to have 512 MiB. The claim is <= PV size.

IOW, a claim of 512 MiB can work with any PV of 512 MiB or more, 1 GiB included.